test/e2e/setup.go

```diff
@@ -245,3 +264,10 @@ var _ = AfterSuite(func() {
 		panic(err)
 	}
 })
+
+var _ = JustAfterEach(func() {
+	gtd := CurrentGinkgoTestDescription()
+	metricName := strings.Replace(gtd.ComponentTexts[1], " ", ".", -1)
+
+	log.WithFields(logrus.Fields{e2emetrics.LogEntryIsE2EEmittableMetric: "true", e2emetrics.LogEntryMetricName: metricName, e2emetrics.LogEntryMetricStatus: !gtd.Failed, e2emetrics.DimensionARMResourceID: resourceIDFromEnv(), e2emetrics.DimensionARMGeoLocation: _env.Location(), e2emetrics.DimensionResourceType: "openShiftClusters"}).Info("Emitting Metric: " + metricName)
```

Comment:
This line is long (374 columns) and difficult to read. Can we break it up to make it more scannable:
```suggestion
	log.WithFields(logrus.Fields{
		e2emetrics.LogEntryIsE2EEmittableMetric: "true",
		e2emetrics.LogEntryMetricName:           metricName,
		e2emetrics.LogEntryMetricStatus:         !gtd.Failed,
		e2emetrics.DimensionARMResourceID:       resourceIDFromEnv(),
		e2emetrics.DimensionARMGeoLocation:      _env.Location(),
		e2emetrics.DimensionResourceType:        "openShiftClusters"},
	).Info("Emitting Metric: " + metricName)
```